samples/certificate_provisioning/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,18 @@
+# WiFi
+CONFIG_WIFI=y
+CONFIG_WIFI_ESP_AT=y
+CONFIG_WIFI_ESP_AT_MDM_RX_BUF_COUNT=40
+
+CONFIG_NET_L2_WIFI_SHELL=y
+CONFIG_GOLIOTH_SAMPLE_WIFI=y
+
+# General config
+CONFIG_HEAP_MEM_POOL_SIZE=4096
```

Comment:
I chose this value arbitrarily from the nRF9160dk conf file. This symbol is necessary for the file system:

```
warning: FILE_SYSTEM_SHELL (defined at subsys/fs/Kconfig:46) was assigned the value 'y' but got the
value 'n'. Check these unsatisfied dependencies: (HEAP_MEM_POOL_SIZE > 0) (=n). See

```

But is there a way to know what size is necessary?

Reply:
I'm thinking maybe we should move this symbol to `prj.conf` too, since all the boards will need the file system shell.